app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -122,6 +126,53 @@
         $node.attr("aria-expanded", state);
       }
 
+      function setSessionStorage() {
+        var isOpenSubsections = $('.subsection--is-open').length;
+        if (isOpenSubsections) {
+          var $openSubsections = $('.subsection--is-open');
+          $openSubsections.each(function(index) {
+            var subsectionOpenContentId = $(this).find('.subsection__content').attr('id');
+            sessionStorage.setItem( subsectionOpenContentId , subsectionOpenContentId);
+          });
+        }
+      }
+
+      function removeSessionStorage() {
+        var isClosedSubsections = $('.subsection').length;
+        if (isClosedSubsections) {
+          var $closedSubsections = $('.subsection');
+          $closedSubsections.each(function(index) {
+            var subsectionClosedContentId = $(this).find('.subsection__content').attr('id');
+            sessionStorage.removeItem( subsectionClosedContentId , subsectionClosedContentId);
+          });
+        }
+      }
+
+      function checkSessionStorage() {
+
+        // Get section IDs from Session Storage
+        var sessionStorageItems = [];
+        for ( var i = 0, len = sessionStorage.length; i < len; ++i ) {
```

Comment:
Session storage is shared across subdomains so if any keys are set elsewhere on GOV.UK then they'll get returned here. Maybe we could iterate over the elements in the DOM and check if they exist in storage instead?